doc/modules/linear_model.rst

```diff
@@ -313,15 +314,74 @@ is correct, i.e. that the data are generated by this model.
 They also tend to break when the problem is badly conditioned
 (more features than samples).
 
-.. figure:: ../auto_examples/linear_model/images/sphx_glr_plot_lasso_model_selection_001.png
-    :target: ../auto_examples/linear_model/plot_lasso_model_selection.html
+.. figure:: ../auto_examples/linear_model/images/sphx_glr_plot_lasso_lars_ic_001.png
+    :target: ../auto_examples/linear_model/plot_lasso_lars_ic.html
     :align: center
     :scale: 50%
 
+Mathematical details
+""""""""""""""""""""
+
+The definition of AIC (and thus BIC) might differ in the literature. In this
+section, we give more information regarding the criterion computed in
+scikit-learn. The AIC criterion is defined as:
+
+.. math::
+    AIC = -2 \ln(\hat{L}) + 2 d
+
+where :math:`\hat{L}` is the maximum likelihood of the model and
+:math:`d` is the number of parameters (as well referred as degrees of
+freedom in the previous section).
+
+For a linear Gaussian model, the maximum log-likelihood is defined as:
+
+.. math::
+    \ln(\hat{L}) = - \frac{n}{2} \ln(2 \pi) - \frac{n}{2} \ln(\hat{\sigma^2}) - \frac{\sum_{i=1}^{n} (y_i - \hat{y_i})^2}{2\hat{\sigma^2}}
+
+where :math:`\hat{\sigma^2}` is an estimator of the noise variance,
+:math:`y_i` and :math:`\hat{y_i}` are respectively the true and predicted
+targets, and :math:`n` is the number of samples.
+
+Plugging the maximum log-likelihood in the AIC formula yields:
+
+.. math::
+    AIC = n \ln(2 \pi \hat{\sigma^2}) + \frac{\sum_{i=1}^{n} (y_i - \hat{y_i})^2}{\hat{\sigma^2}} + 2d
+
+The left-hand term of the above expression is sometimes discarded since it is a
+constant and does not change the rank of the models evaluated. In addition,
+it is sometimes stated that the AIC is equivalent to the :math:`C_p` statistic
+[12]_. However, this is important to note that it is up to some constant
+and factor term.
```

Comment:
```suggestion
The first term of the above expression is sometimes discarded since it is a
constant when :math:`\hat{\sigma}^2` is provided. In addition,
it is sometimes stated that the AIC is equivalent to the :math:`C_p` statistic
[12]_. However, this is important to note that it is up to some constant
and factor term.
```

this last sentence is mysterious without a reference